test/integration/full/landmark-no-more-than-one-main/frames/level2.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html id="pass4">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>This should pass, too</p>
```

Comment:
It passes because this particular document does not have a main landmark. A document will pass under this rule as long as it has either just one main landmark or none at all.